frontend/e2e/fixtures/cycleTimeByStatus/cycleTimeByStatusFixture.ts

```diff
@@ -0,0 +1,141 @@
+export const cycleTimeByStatusFixture = {
+  projectName: 'Heartbeat Metrics',
+  dateRange: {
+    startDate: '2024-03-27T00:00:00.000+08:00',
+    endDate: '2024-03-29T23:59:59.999+08:00',
+  },
+  calendarType: 'Calendar with Chinese Holiday',
+  metrics: [
+    'All',
+    'Velocity',
+    'Cycle time',
+    'Classification',
+    'Rework times',
+    'Lead time for changes',
+    'Deployment frequency',
+    'Dev change failure rate',
+    'Dev mean time to recovery',
+  ],
+  board: {
+    type: 'Jira',
+    boardId: '31',
+    email: 'heartbeatuser2023@gmail.com',
+    projectKey: 'ADM',
+    site: 'dorametrics',
+    token: process.env.E2E_TOKEN_JIRA,
+  },
+  pipelineTool: {
+    type: 'BuildKite',
+    token: process.env.E2E_TOKEN_BUILD_KITE,
+  },
+  pipelineCrews: ['heartbeat-user', 'Unknown'],
+  sourceControl: {
+    type: 'GitHub',
+    token: process.env.E2E_TOKEN_GITHUB,
+  },
+  crews: ['heartbeat user', 'Chao Wang', 'Junbo Dai'],
+  assigneeFilter: 'lastAssignee',
+  cycleTime: {
+    type: 'byColumn',
+    jiraColumns: [
+      {
+        TODO: 'To do',
+      },
+      {
+        Doing: 'In Dev',
+      },
+      {
+        Blocked: 'Block',
+      },
+      {
+        Review: 'Review',
+      },
+      {
+        'READY FOR TESTING': 'Waiting for testing',
+      },
+      {
+        Testing: 'Testing',
+      },
+      {
+        Done: 'Done',
+      },
+    ],
+    treatFlagCardAsBlock: true,
+  },
+  cycleTimeByStatus: {
```

Comment:
As far as I know, cycleTimeByStatus is not a standard key(still `cycleTime` after saved json?) of the saved file.
We kept these kind of files here in order to test the fixtures against the saved file.

So could this be refined anyway?